test/hotspot/jtreg/TEST.ROOT

```diff
@@ -32,7 +32,7 @@
 # intermittent:         flaky test, known to fail intermittently
 # randomness:           test uses randomness, test cases differ from run to run
 # cgroups:              test uses cgroups
-keys=stress headful intermittent randomness cgroups
+keys=stress headful intermittent randomness cgroups flag-sensitive
```

Comment:
Maybe add a comment above about the key, similar to the others. Something like: 
```
# flag-sensitive:       test is sensitive to certain flags and might fail when flags are passed in
```

Reply:
Thanks, I reworded it just slightly